scripts/ci/jobs/ocp_qa_e2e_tests.py

```diff
@@ -15,4 +15,10 @@
 os.environ["ROX_RISK_REPROCESSING_INTERVAL"] = "15s"
 os.environ["ROX_SENSOR_CONNECTION_RETRY_MAX_INTERVAL"] = "30s"
 
+for v in ['JOB_NAME_SAFE', 'CLUSTER_FLAVOR_VARIANT', 'CLUSTER_PROFILE_NAME', 'CLUSTER_TYPE', 'OPENSHIFT_CI_STEP_NAME']:
+    print(v + ':' + os.environ.get(v, ''))
```

Comment:
FYI, there is a `debug.txt` artifact for most steps that dumps the env. https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/stackrox_stackrox/10240/pull-ci-stackrox-stackrox-master-ocp-4-11-qa-e2e-tests/1765178781493891072/artifacts/qa-e2e-tests/ocp-4-create/artifacts/debug.txt